src/modules/reserve-overview/ReserveActions.tsx

```diff
@@ -132,7 +134,8 @@ export const ReserveActions = ({ underlyingAsset }: ReserveActionsProps) => {
   const maxAmountToSupply = getMaxAmountAvailableToSupply(
     balance.amount,
     poolReserve,
-    underlyingAsset
+    underlyingAsset,
+    poolComputed.minRemainingBaseTokenBalance
   ).toString();
```

Comment:
In all these cases you can do `const { poolComputed: { minRemainingBaseTokenBalance } } = useRootStore()` to be even cleaner when using it.

Reply:
✔️